test/Examples/PlaneCurve-test.jl

```diff
@@ -114,9 +114,15 @@ end
 end
 
 @testset "ProjPlaneCurve constructors" begin
+<<<<<<< HEAD
     R, (x, y, z) = PolynomialRing(QQ, ["x", "y", "z"])
     T, _ = grade(R)
     F = T(y^3 * x^6 - y^6 * x^2 * z)
+=======
+    R, (x,y,z) = PolynomialRing(QQ, ["x", "y", "z"])
+    T, _ = grade(R)
+    F = T(y^3*x^6 - y^6*x^2*z)
+>>>>>>> Minor changes to Plane Curve and Curve Divisor
```

Comment:
Here a merge conflict was not properly resolved

Reply:
I probably forgot to save the file after removing these lines...